nss-day-cohort-47/reactive-nutshell-three-https-in-a-pod
# Hp edit article

# Description
Implemented edit functionality for a user to be able to edit their published articles

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Navigate to articles page, post a new article, select edit and update any field, select submit, article should return updated object.
# Checklist:
